html/index.html.in

```diff
@@ -73,15 +75,15 @@
         <link rel="manifest" href="./manifest.json">
     </head>
     <body>
-        <span class="fa hide" id="faChecker"></span>
+        <span class="fa" id="faChecker" style="display: none;"></span>
         <script type="text/javascript">
             if ($('#faChecker').css('fontFamily') !== 'FontAwesome') {
                 $('<link type="text/css" rel="stylesheet" href="./css/font-awesome.min.css">').appendTo('head');
                 console.warn('FontAwesome CSS from CDN not available - reverting to local copy');
             }
         </script>
 
-        <span class="pull-right flip hide" id="rtlChecker"></span>
+        <span class="float-right flip" id="rtlChecker" style="display: none;"></span>
         <script type="text/javascript">
             if ($('#rtlChecker').css('float') !== 'left' && !$('#rtlStylesheet').attr('disabled')) {
```

Comment:
@sushain97 This thing causes the error. When its loaded initially `!$('#rtlStylesheet').attr('disabled')` this statement isnt satisfied as `localizeinteface()` js function, which changes the `#rtlStylesheet` attr doesnot happen when changing from one locale to another locale. When Its reloaded the LocalizeInteface function runs and the statement statisfies and the rtl css is loaded.